test/acl.test.js

```diff
@@ -48,22 +44,28 @@ describe('security scopes', function() {
     testModel.attachTo(ds);
   });
 
-  it('should allow access to models for the given scope by wildcard', function() {
+  it('should allow access to models for the given scope by wildcard', function(done) {
     Scope.create({name: 'userScope', description: 'access user information'},
       function(err, scope) {
         ACL.create({
           principalType: ACL.SCOPE, principalId: scope.id,
           model: 'User', property: ACL.ALL,
           accessType: ACL.ALL, permission: ACL.ALLOW,
         }, function(err, resource) {
-          Scope.checkPermission('userScope', 'User', ACL.ALL, ACL.ALL, checkResult);
-          Scope.checkPermission('userScope', 'User', 'name', ACL.ALL, checkResult);
-          Scope.checkPermission('userScope', 'User', 'name', ACL.READ, checkResult);
+          async.parallel([
+            clbk => Scope.checkPermission('userScope', 'User', ACL.ALL, ACL.ALL, clbk),
+            clbk => Scope.checkPermission('userScope', 'User', ACL.ALL, ACL.ALL, clbk),
+            clbk => Scope.checkPermission('userScope', 'User', 'name', ACL.ALL, clbk),
+            clbk => Scope.checkPermission('userScope', 'User', 'name', ACL.READ, clbk),
+          ], (err) => {
+            assert(!err);
```

Comment:
While you are changing this part: let's improve the assertion to produce a more helpful error message.

`assert(!err)` say something like `expected false to be true` and hide all details about the error.

Let's use the following:

```js
assert.ifError(err);
done();
```